apis/compute/v1alpha3/types.go

```diff
@@ -154,7 +142,6 @@ type EKSClusterParameters struct {
 	// https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html
 
 	// Region for this EKS Cluster.
-	// +kubebuilder:validation:Enum=us-west-2;us-east-1;eu-west-1
 	Region EKSRegion `json:"region"`
```

Comment:
Since `EKSRegion` is no longer an enum and just a plain string do you want to change `Region` type to `string`?

Reply:
done